light-zk.js/src/transaction/prover.ts

```diff
@@ -122,7 +122,7 @@ export class Prover {
     this.vKey = vKey;
   }
 
-  async verify(proof?: proofData): Promise<boolean> {
+  async verify(_proof?: proofData): Promise<boolean> {
```

Comment:
can you replace line 131 with the following as we discussed in https://github.com/Lightprotocol/light-protocol/pull/150 ?
```
proof || this.proof,
```

Reply:
done ✅